src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj

```diff
@@ -5,6 +5,7 @@
     <Description>Monitor.Tests Class library</Description>
     <PackageId>Monitor.Tests</PackageId>
     <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <TargetFrameworks>net452</TargetFrameworks>
```

Comment:
BTW: I can only build from VS 2017, not from the command line, nor from the root of the repo.
Is that expected?
Do I need to use any special argument to build from the command line?